spec/14-classes.md

```diff
@@ -58,7 +58,15 @@ object. As such, assignment of a handle does not copy the object itself.
 
 <pre>
   <i>class-declaration:</i>
-    <i>class-modifier<sub>opt</sub></i>  class  <i>name   class-base-clause<sub>opt</sub>  class-interface-clause<sub>opt</sub></i>   {   <i>trait-use-clauses<sub>opt</sub>   class-member-declarations<sub>opt</sub></i> }
+    <i>class-modifier<sub>opt</sub></i>  class  <i>name   class-base-clause<sub>opt</sub>  class-interface-clause<sub>opt</sub></i>   {   <i>class-members<sub>opt</sub></i>   }
+    
+   <i>class-members:</i>
+    <i>class-member</i>
+    <i>class-members   class-member</i>
+    
+   <i>class-member:</i>
+    <i>trait-use-clauses</i>
+    <i>class-member-declarations</i>
```

Comment:
Fair enough about keeping trait/class members separate. But going back to the original question, I still think that `trait-use-clauses` can be added to both `class-member-declarations` and  `trait-member-declarations`, rather than adding additional `class-members` (and presumably also `trait-members`) productions. So something like the following:

<pre>
  <i>class-member-declarations:</i>
    <i>class-member-declaration</i>
    <i>class-member-declarations   class-member-declaration</i>

  <i>class-member-declaration:</i>
    <i>trait-use-clauses</i>
    <i>const-declaration</i>
    <i>property-declaration</i>
    <i>method-declaration</i>
    <i>constructor-declaration</i>
    <i>destructor-declaration</i>

  <i>trait-member-declarations:</i>
    <i>trait-member-declaration</i>
    <i>trait-member-declarations   trait-member-declaration</i>

  <i>trait-member-declaration:</i>
    <i>trait-use-clauses</i>
    <i>property-declaration</i>
    <i>method-declaration</i>
    <i>constructor-declaration</i>
    <i>destructor-declaration</i>
</pre>

Instead of:

<pre>
  <i>class-members:</i>
    <i>class-member</i>
    <i>class-members   class-member</i>
    
  <i>class-member:</i>
    <i>trait-use-clauses</i>
    <i>class-member-declarations</i>

  <i>class-member-declarations:</i>
    <i>class-member-declaration</i>
    <i>class-member-declarations   class-member-declaration</i>

  <i>class-member-declaration:</i>
    <i>const-declaration</i>
    <i>property-declaration</i>
    <i>method-declaration</i>
    <i>constructor-declaration</i>
    <i>destructor-declaration</i>

  <i>trait-members:</i>
    <i>trait-member</i>
    <i>trait-members   trait-member</i>
    
  <i>trait-member:</i>
    <i>trait-use-clauses</i>
    <i>trait-member-declarations</i>

  <i>trait-member-declarations:</i>
    <i>trait-member-declaration</i>
    <i>trait-member-declarations   trait-member-declaration</i>

  <i>trait-member-declaration:</i>
    <i>property-declaration</i>
    <i>method-declaration</i>
    <i>constructor-declaration</i>
    <i>destructor-declaration</i>
</pre>

Or am I still missing some distinction here?